rero_ils/modules/libraries/api.py

```diff
@@ -241,7 +244,7 @@ def is_open(self, date=None, day_only=False):
         #   is into this periods (depending of `day_only` method argument).
         day_name = date.strftime("%A").lower()
         regular_rule = [
-            rule for rule in self['opening_hours']
+            rule for rule in self.get('opening_hours', [])
```

Comment:
For me this should be part of the library validation process. We shouldn't define a "open" exception date without any opening hours --> this should raise an exception into `extended_validation` method.

Reply:
`opening_times` is not required in the jsonschema and should not be so we cannot assume it will be there. Therefore use get.